pkg/session_recording/recordings.jsx

```diff
@@ -176,14 +186,20 @@
             if (!r) {
                 return null;
             }
+            let spawn = [
+                "/usr/bin/tlog-play",
+                "--persist",
+                "--follow",
+                "--reader=journal",
+                "-M", "TLOG_REC=" + r.id,
+                "--speed=" + Math.pow(2, this.state.speed_exp),
+            ];
+            if (this.state.paused === true) {
```

Comment:
I would say this is excessive, `if (this.state.paused) {` would be just fine :).

Reply:
Agree 👍 Fixed.